Gemfile

```diff
@@ -6,7 +6,7 @@ group :development, :test do
     gem "rubocop"
 end
 
-gem "appwrite", "~> 6.0.0"
+gem "appwrite", "~> 7.0.0"
```

Comment:
Just double checking, does this mean it will pull 7.1?

Reply:
Updated now to use >=7.1 < 7.2